exotica_python/CMakeLists.txt

```diff
@@ -16,4 +16,6 @@ include_directories(
 )
 
 pybind_add_module(pyexotica MODULE src/Exotica.cpp)
-install(TARGETS pyexotica LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION})
+install(TARGETS pyexotica LIBRARY DESTINATION ${CATKIN_GLOBAL_PYTHON_DESTINATION})
+catkin_install_python(PROGRAMS scripts/example_aico.py scripts/example_ompl.py scripts/example_ik.py scripts/publish_trajectory.py DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Comment:
Perhaps a submodule, i.e. ``from pyexotica.tools import publish_trajectory``. So ``pyexotica`` is the ``.so`` and ``tools`` a folder inside the module folder?